cluster/log-dump/log-dump.sh

```diff
@@ -104,7 +104,7 @@ function log-dump-ssh() {
 function copy-logs-from-node() {
     local -r node="${1}"
     local -r dir="${2}"
-    local files=( ${3} )
+    read -r -a files <<< "${3}"
```

Comment:
shellcheck detects SC2206.
```
In ./cluster/log-dump/log-dump.sh line 107:
    local files=( ${3} )
                  ^--^ SC2206: Quote to prevent word splitting, or split robustly with mapfile or read -a.
```